src/customize-pop/gnome-tweaks-extensions.md

```diff
@@ -0,0 +1,202 @@
+
+![Select Startup App](/images/gnome-tweaks-extensions/select-startup-app.png)
+
+### Top Bar
+
+`Top Bar` settings allow you to configure the clock to display additional information.
+
+![Top Bar Tweaks](/images/gnome-tweaks-extensions/top-bar-tweaks.png)
+
+You can also choose to display week numbers in the system calendar application.
+
+![Calendar Tweaks](/images/gnome-tweaks-extensions/calendar-tweaks.png)
+
+### Window Titlebars
+
+These settings allow you to interact with windows using mouse clicks. You can also choose available titlebar buttons and their placement. Right, middle, and left-click buttons can each be configured to perform one of the following actions when clicking a window's titlebar:
+
+![Titlebar Tweaks](/images/gnome-tweaks-extensions/titlebar-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Lower | A "send to back" action occurs. |
+| Menu | Display a window manipulation menu. |
+| Minimize | Minimize a window.|
+| Toggle Maximize | Maximize a window, the window returns to its previous size on second click. |
+| Toggle Maximize Horizontally* | Extend a window to the left and right sides of the screen without modifying vertical dimensions. |
+| Toggle Maximize Vertically* | Expand a window to the top and bottom of the screen without modifying horizontal dimensions. |
+| Toggle Shade* | Collapse or expand a window from its titlebar when the window's titlebar is clicked. |
+
+\* This feature is not supported by all application windows.
+
+### Windows
+
+The `Windows` menu offers additional features for interacting with system and application windows.
+
+![Windows Tweaks](/images/gnome-tweaks-extensions/windows-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Attach Modal Dialogs | Modal dialogs (such as windows that spawn when performing a "Save as...") stay in a fixed position within their parent window. |
+| Center New Windows | New windows launch perfectly centered on the screen. |
+| Resize with Secondary-Click | Maximize or returne a window to its previous size by double-clicking the primary mouse button. |
+| Window Action Key | Drag windows freely by holding this key, left-clicking anywhere in the window, and dragging with the mouse. Resize windows by holding this key, right-clicking anywhere in a window, and dragging the mouse. |
+| Window Focus | Set the action that brings a window into focus: <ul><li>Click to Focus: Click a window to bring it into focus.</li><li>Focus on Hover: Hover the cursor over a window to bring it into focus. Focus is maintained if the cursor is moved to the desktop.</li><li>Secondary-Click: Hover the cursor over a window to bring it into focus. Focus is not maintained if the cursor is moved to the desktop.</li><li>Raise Windows When Focused: Focus a window to bring it to the front. This option is available when either `Focus on Hover` or `Secondary-Click` are chosen. |
```

Comment:
The last item should not be part of the same list as the first three.

For the first three, "Choose how windows are focused" or "Choose how the mouse focuses windows" might be more clear than "set the action," since I don't consider hovering the cursor quite as much of an action as clicking.

(Note for posterity, I thought the Secondary-Click option's description sounded weird, but there is an [open GNOME GitLab issue](https://gitlab.gnome.org/GNOME/gnome-tweaks/-/issues/369) regarding that name.)

Reply:
Added last item into its own row, updated with suggested wording, and yea that name makes no sense.